attributes/default.rb

```diff
@@ -61,3 +61,42 @@
 default["confluent"]["schema-registry"]["log4j.properties"]["log4j.appender.ROLLINGFILE.MaxBackupIndex"] = "10"
 default["confluent"]["schema-registry"]["log4j.properties"]["log4j.appender.ROLLINGFILE.layout"] = "org.apache.log4j.PatternLayout"
 default["confluent"]["schema-registry"]["log4j.properties"]["log4j.appender.ROLLINGFILE.layout.ConversionPattern"] = "[%d] %p %m (%c:%L)%n"
+
+default["confluent"]["kafka-connect"]["jar_urls"] = []
+# Expects a hash where they keys are the file name and the values are hashes representing the property_name:value pairs
+default["confluent"]["kafka-connect"]["properties_files"] = {}
+
+default["confluent"]["kafka-connect"]["distributed_mode"] = true
+default["confluent"]["kafka-connect"]["worker_properties_file_name"] = "worker.properties"
```

Comment:
Well its not anymore work now that its done I think. Do you think it would ever be useful to be able to change the name of the generated file? The only thing I was thinking was to avoid clashes if for some reason the consumer uses a newer version of confluent that comes with a worker.properties and didn't want to override it. I can remove it though if that isn't useful